styleguide/_themes/derek/scss/_main.scss

```diff
@@ -5,7 +5,7 @@
 @import 'globalElements/mainnav';
 @import 'globalElements/rug';
 @import 'globalElements/footer';
-@import 'globalElements/basement';
+@import '../../../derek/global-components/basement/basement';
```

Comment:
in the future can we create a baseURL for this so the paths dont get so out of hand?